_projects/git_task_list.md

```diff
@@ -0,0 +1,87 @@
+---
+collaborating_projects:
+  - coala
+  - meta-review
+desc: "List pull request/merge request (in gitlab) that have no reviewer yet. Provide an easy way for new comer / developer to do contribution to a project by giving review to other people pull requests."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
+  - https://github.com/coala/coala/issues/3615
+  - https://github.com/coala/coala/issues/4080
+markdown: git_task_list.md
+mentors:
+  - jayvdb
+  - yukiisbored
+  - yashovardhanagrawal
+  - sims1253
+name: "Git Task List"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "Familiarity with Git, GitHub API Gerrit API, Phabricator API, and GitLab API"
+  - "Familiarity with EmberJS, Javascript, and GitHub Pages"
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The basic idea is an extended version of [GSOC Task List][2] which was initialy
+intended for GSOC Applicans who don't have to a little experience in
+open-source. Hence, [that][1] project will be the starting point of this
+project.
+
+This project will help sharing the tasks of reviewing a pull request on a
+project. This is also maybe the solution of https://github.com/coala/coala/issues/4080,
```

Comment:
issue links belong in the project metadata, and should not be listed in the project spec.

The project spec should use sentences which explain the problems and solutions.